docs/how_to_guides/how_to_use_a_property_model.rst

```diff
@@ -70,6 +70,12 @@ A portion of the displayed output is shown below.
 
 .. testoutput::
 
+   WARNING: model contains export suffix 'fs.state_block[0].scaling_factor' that
+   contains 4 component keys that are not exported as part of the NL file.
+   Skipping.
+   WARNING: model contains export suffix 'fs.state_block[0].scaling_factor' that
+   contains 4 component keys that are not exported as part of the NL file.
+   Skipping.
```

Comment:
This looks like something we wouldn't want showing up in the how-to

Reply:
1. It's just in the test -- so the reader won't see it.
2. I talked with @andrewlee94 about this a few weeks ago and there is not a workaround nor is one planned (IDAES and Pyomo are implementing a new model scaling framework)